src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -0,0 +1,95 @@
+"""
+Annotate VCF files with VRS
+
+Input Format: VCF
+Output Format: VCF
+
+"""
+
+import argparse
+import hashlib
+import logging
+import sys
+import pickle
+import time
+import coloredlogs
+from biocommons.seqrepo import SeqRepo
+from pysam import VariantFile
+from ga4gh.core import ga4gh_identify
+from ga4gh.vrs import models
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+from ga4gh.vrs.extras.translator import Translator
+
+dp = SeqRepoDataProxy(SeqRepo("/usr/local/share/seqrepo/latest"))
+tlr = Translator(data_proxy=dp)
+
+_logger = logging.getLogger()
+
+INFO_FIELD_ID = "VRS_Allele"
+vrs_data = {}
+
+
+def parse_args(argv):
+    ap = argparse.ArgumentParser()
+    ap.add_argument("VCF_IN")
+    ap.add_argument("--out", "-o", default="-")
+    ap.add_argument("--failure-probability", "-f", type=float, default=0)
+    ap.add_argument("--vrs_file", default="-")
+    opts = ap.parse_args(argv)
+    return opts
+
+
+def record_digests(record):
```

Comment:
It'd be nice to see some tests

Reply:
Can you expand on this for me please?

Reply:
Ya, so I would recommend adding tests to ensure that the actual functionality matches the expected functionality. It's good practice to create tests!